### ISSUE TITLE
Upload server emojis and art

### ISSUE THREAD

**Issue by Vyvy-vi** [gurkult/branding#3]
### Description:
Currently, the branding repo is pretty much empty, perhaps we could add the files here so that it could be open for contribution(and display).

### Proposed folder structure:
```
├── emotes
│   ├── base_emojis
│   │   └── gurkmoji_base.svg
│   └── custom_emojis
├── community-logos
│   ├── archive
│   └── current
├── Legal
│   ├── licencing.md
│   └── acknowledgements.md
├── slogans-and-lore
├── LICENSE
├── CONTRIBUTING.md
└── README.md
```

After this, we could add the emotes inside the repo. Perhaps, we could also try to upload the source files(making them easily editable, and faster to contribute to). 
Thoughts? (Is there unnecessary stuff in this folder structure?)

#### More context: [link-to-relevant-discord-chat](https://discord.com/channels/789192517375623228/789193817051234306/794579139314712592)

**Comment by gustavwilliam** [comment]
Yeah, it would be great to get some structure here.

**Comment by uncomfyhalomacro** [comment]
@Vyvy-vi Your proposed structure is clean and intuitive. I do not think there are more to comment but to focus more on what the contents are for each directory. What do you think @gustavwilliam ?

**Comment by gustavwilliam** [comment]
I think that it works, generally. There are just a few things we could refine.

I don't see a reason to include a `Legal` folder. We could put attributions and license-stuff in the README.

`community-logos` should just be called `logos` imo. What is the thought behind including an archived section? We have a hit history, so everything we used in the past should be available there.

We can also have sub-directories for the logos. Here are a few options. We can add new sub-folders as we need them.
- `logos/color`
- `logos/bw`

What's up with the sub-directories inside `emotes`? We could just have `emotes/category`, where some examples of categories are:
- Gurkmoji
- Other

We could also have an `icons` category, for icons used by the bot.

**Comment by Vyvy-vi** [comment]
there's a sub-directory for emoji bases, so that new contributors could pick them up easily, rather than going through the whole gurkmoji list.

Yes, `logos` would be better and shorter name :+1:
The `logos/archive` is supposed to hold old community logos(in case we change logos in the future and the `current` sub-directory holds the current ones in use)

I thought about a `Legal` folder, since this would be opensource art, we might need to write some extra policies etc. especially when contributors edit the existing art, after getting inspired from other art. (I'm not sure about what all legal stuff goes into this)

**Comment by gustavwilliam** [comment]
For the archive, we already have fit history if we want the previous logos. I personally don’t see a reason to keep the outdated logos. Maybe @ReneganRonin or someone else has a third view on this?

As for `legal`, I feel kind of the same way, where I’m not sure if we really need it. If we have a gurkmoji folder, we could probably just shove a license inside the folder, or put it in the readme.

**Comment by Akarys42** [comment]
> For the archive, we already have fit history if we want the previous logos. I personally don’t see a reason to keep the outdated logos. Maybe ReneganRonin or someone else has a third view on this?

I don’t see any reason for it, we have the git history. 

> As for `legal`, I feel kind of the same way, where I’m not sure if we really need it. If we have a gurkmoji folder, we could probably just shove a license inside the folder, or put it in the readme.

I agree, the attributions can just be in the README for third party data.

**Comment by uncomfyhalomacro** [comment]
```
├── emotes
│   ├── base_emojis
│   │   └── gurkmoji_base.svg
│   └── custom_emojis
├── community-logos
│   ├── archive? 
│   └── current?
├── ACKNOWLEDGEMENTS.md
├── slogans-and-lore
├── LICENSE
├── CONTRIBUTING.md
└── README.md
```
I still like the idea of an archive because even if we have git history, i think it will be a hassle for someone to look for an old svg or image through git history. This is a modified structure @gustavwilliam, what do you think?

**Comment by Vyvy-vi** [comment]
Maybe, we could replace the archive folder, with a `ARCHIVE.md` file, that links to the old logos(that might be used by contributors, for inspiration, when making new ones), probably stored in gists/drive/cloud/discord/img-dump/something?

**Comment by Akarys42** [comment]
Maybe we can slap some instructions on the README about how to inspect the
history, but eh, that isn't hard.

**Comment by gustavwilliam** [comment]
Yes, exactly. We should probably do that, if there’s a need for it @Akarys42

**Comment by Vyvy-vi** [comment]
Yes, I think we could add that in the README :+1:
Also, I think maybe we could link to the gurkan_base emoji, from the CONTRIBUTING.md, rather than having a directory for it 🤔
